website/project/model.py

```diff
@@ -2173,6 +2186,7 @@ def fork_node(self, auth, title='Fork of '):
         # Clear permissions before adding users
         forked.permissions = {}
         forked.visible_contributor_ids = []
+        forked.save()
```

Comment:
I considered this already, but I could change those two signaled functions to be more error-resistant; so that if a contributor is added before the `mailing_node_created` signal is emitted it creates a subscription and adds that contributor to it, and when that signal is sent it looks for an existing subscription before creating a new one. This could obviate the `mailing_node_created` signal entirely, but doesn't guarantee that we would know if any subscriptions were configured improperly.